src/main/java/roboy/dialog/states/ordinaryStates/IntroductionState.java

```diff
@@ -37,18 +45,25 @@
  * 3) No parameters are used.
```

Comment:
this is wrong
now, parameters are used!

Reply:
Fixed